src/stylesheets/components/_highlight.scss

```diff
@@ -28,9 +28,11 @@
   padding: ($base-spacing-unit + 1) 0;
 
   p, li {
-    line-height: 1.125; // 18px;
+    @include fontsize(14);
```

Comment:
Leave line-heights and use the `milli` mixin and update the mixin from:

```
@mixin milli {
  @include fontsize(14);
  @include media($tablet) {
    @include fontsize(16);
  }
}
```
To this:
```
@mixin milli {
  @include fontsize(14);
  @include media($desktop) {
    @include fontsize(16);
  }
}